fe/hive-udf/pom.xml

```diff
@@ -17,6 +17,7 @@
         <dependency>
             <groupId>io.trino.hive</groupId>
```

Comment:
This package is not actually used, maybe can be removed.

Reply:
UDF needs to implement some interfaces defined by Hive, so it cannot be removed.